lang/rs/Cargo.toml

```diff
@@ -11,3 +11,6 @@ members = [
     "examples/fwd"
 ]
 
+[profile.release]
+lto = true
```

Comment:
What is the performance gain when using this feature?

Reply:
I've noticed 10-15% drop in performance if lto is not enabled with Rust builds.